### ISSUE TITLE
feature: more interactive nav link

### ISSUE THREAD

**Issue by CodeTorso** [OpenCodeEra/OpenCodeEra#17]
I have this cool nav link idea.

I could improve it for you. 

my plan is to bring the nav link little bit down and after the user moves little bit down the nav link will go up and display will then turn sticky and the background of nav-link should have alpha value 80. That is they should have 50% transparency and their size should also be reduced somewhat to give a larger space for much of the website.

**Comment by CodeTorso** [comment]
Hey @AdarshRawat1, Do you think we should make this website using react?? React has a huge community of contributors which we currently don't have access to. What do you think ? I have just learnt react looking for your response.

**Comment by CodeTorso** [comment]
actually I am still learning react

**Comment by AdarshRawat1** [comment]
> Hey @AdarshRawat1, Do you think we should make this website using react?? React has a huge community of contributors which we currently don't have access to. What do you think ? I have just learnt react looking for your response.

I think we should start migrating from scratch in a new repository.
Your wthoughts @CodeTorso @AKACHI-4 @s-dabral

**Comment by s-dabral** [comment]
Yes @AdarshRawat1, if we're gonna make the website using react, we should start new repository. 
With this, we can continue using old repository as the website until the new one is complete.

**Comment by AKACHI-4** [comment]
Yah sure that would be great 👍

**Comment by CodeTorso** [comment]
@AdarshRawat1 Wanna start the new repo??

**Comment by CodeTorso** [comment]
You may roll it out for HacktoberFest since our website is also about that.

Many small contributors will join us.

What do you think?

**Comment by AdarshRawat1** [comment]
> You may roll it out for HacktoberFest since our website is also about that.
> 
> Many small contributors will join us.
> 
> What do you think?

Sounds great

**Comment by CodeTorso** [comment]
Can someone like I be maintainer ?? Just asking. 🤔🤔. 

I will still be contributing though even If I am not.

I was asking this since I looked at your Readme.md file

**Comment by AdarshRawat1** [comment]
> Can someone like I be maintainer ?? Just asking. 🤔🤔.
Yes @CodeTorso you can be a maintainer of a project that you are familiar with. As you have contributed to Open Code Era's website, you can be a maintainer in this Repository. 

> I will still be contributing though even If I am not.
Appriciated!

> I was asking this since I looked at your Readme.md file.

If you plan to maintain any other repo you'll have to solve issues, and then we can promote you.

**Comment by CodeTorso** [comment]
Ok let me make 4 PRs then get me as maintainer in this Repo. 

I don't want those 4 PRs to be merged by myself.

**Comment by AdarshRawat1** [comment]
> Ok let me make 4 PRs then get me as maintainer in this Repo.
> 
> I don't want those 4 PRs to be merged by myself.

Sure, till then you can open issues and make your valuable contributions via PR.

**Comment by CodeTorso** [comment]
Can you check all the PRs and merge them at 4-5 am or 6am. Under HacktoberFest label

**Comment by AdarshRawat1** [comment]
> Can you check all the PRs and merge them at 4-5 am or 6am. Under HacktoberFest label

If they are relevant and don't cause any "merge conflicts", then I'll merge them.

**Comment by CodeTorso** [comment]
ok.

Why the hell is this conversation going between these 2 issues😂😂

**Comment by AdarshRawat1** [comment]
> ok.
> 
> Why the hell is this conversation going between these 2 issues😂😂

You are the one writing here and there🤣, I'm just trying to keep up😅.
Actually I'm listing new issues for Hacktoberfest. So, every notification gets me thinking that it's a comment on an issue that needs my reply.

**Comment by AdarshRawat1** [comment]
@CodeTorso Help me list those issues if you are not currently working on the PR.

**Comment by AdarshRawat1** [comment]
> @CodeTorso Help me list those issues if you are currently not working on the PR.

**Comment by CodeTorso** [comment]
ok

**Comment by CodeTorso** [comment]
Can you create a new repo in the organisation for me to start or should create a repo on this very repo

**Comment by CodeTorso** [comment]
At the end this repo also needs a restart

**Comment by CodeTorso** [comment]
> Can you create a new repo in the organisation for me to start or should create a repo on this very repo

I can make 10-15 issues on this new scratch repo if you agree

**Comment by CodeTorso** [comment]
OR do you want OpenCodeEra 2 : The complete webApp

**Comment by CodeTorso** [comment]
that seems a cool name

**Comment by AdarshRawat1** [comment]
> At the end this repo also needs a restart

Let's leave this repo as it is and start from a new repository.
Any suggestions on the name of the repo? <Needs to be related to open code era as we'll be using cloud flare for production hosting and `.dev` domain is provided on the name of repository >

**Comment by AdarshRawat1** [comment]
> that seems a cool name



> OR do you want OpenCodeEra 2 : The complete webApp

Zing!!

**Comment by CodeTorso** [comment]
> Zing!!

First it will be plain HTML CSS JAVA

Then slowly shift to React 

Then Next.js 

Next.js can hugely improve load time.

**Comment by AdarshRawat1** [comment]
> Next.js can hugely improve load time.

sure thing !

Lets name the new repo - Open Code

**Comment by CodeTorso** [comment]
ok

**Comment by CodeTorso** [comment]
Can you create or Should I ?

**Comment by AdarshRawat1** [comment]
> Can you create or Should I ?

I can create it, but you just start it as you see fit, and then we can list those issues for Hacktoberfest.

**Comment by CodeTorso** [comment]
I am starting to write code.

Create the repo. I will create some PRs by the morning.

Also some issues for that PR to other to work upon.

I am naming it OpenCode. You may change it later

**Comment by AdarshRawat1** [comment]
> I am starting to write code.

🙌
> Also some issues for that PR to other to work upon.

Don't forget this repository while you are working on the new one 😏
> I am naming it OpenCode. You may change it later.

we'll discuss this later

**Comment by CodeTorso** [comment]
> Don't forget this repository while you are working on the new one

This was my first prominent open source project.

How can I forget it. Ok bye

**Comment by AdarshRawat1** [comment]
@CodeTorso  Can we please have an update on the PRs?

**Comment by CodeTorso** [comment]
I had some issues.

But the project is running from yesterday. I will be making many PRs today.
